cpu/native/startup.c

```diff
@@ -121,6 +119,12 @@ static const char short_opts[] = ":hi:s:deEoc:"
 #endif
     "";
 
+#if __GLIBC__
+static const bool _is_glibc = true;
+#else
+static const bool _is_glibc = false;
+#endif
```

Comment:
This may look like a good use case for `IS_ACTIVE()`. However, `IS_ACTIVE` only works if the macro is either `1`, `0`, or undefined. `__GLIBC__` however is `2` on current glibc based systems, so it just won't work.